doc/Architecture/arc-tec_x-road_technologies.md

```diff
@@ -60,31 +61,36 @@ See X-Road terms and abbreviations documentation \[[TA-TERMS](#Ref_TERMS)\].
- PAM                | X                   | X                  |                         |
- Liquibase          | X                   | X                  |                         | X
- systemd            | X                   | X                  | X                       | X
- PKCS \#11\[[2](#Ref_2)\]       | X                   | X                  | X                       |
- Dropwizard Metrics | X                   |                    |                         | X
+ **Technology**                     | **Security server** | **Central server** | **Configuration proxy** | **Operational Monitoring Daemon**
+----------------------------------- | ------------------- | ------------------ | ----------------------- | -------------------
+ Java 8                             | X                   | X                  | X                       | X
+ C                                  | X                   | X                  |                         |
+ Logback                            | X                   | X                  | X                       | X
+ Akka 2.X                           | X                   | X                  | X                       | X
+ Jetty 9                            | X\[[3](#Ref_3)\]    | X                  |                         |
+ JRuby 9.X                          |                     | X                  |                         |
+ Ubuntu 18.04                       | X                   | X                  | X                       | X
+ Red Hat Enterprise Linux 7 (RHEL7) | X                   |                    |                         | X
+ PostgreSQL 9.4                     |                     | X\[[1](#Ref_1)\]   |                         |
+ PostgreSQL 10                      | X                   | X                  |                         | X
+ nginx                              |                     | X                  | X                       |
+ PAM                                | X                   | X                  |                         |
+ Liquibase                          | X                   | X                  |                         | X
+ systemd                            | X                   | X                  | X                       | X
+ PKCS \#11\[[2](#Ref_2)\]           | X                   | X                  | X                       |
+ Dropwizard Metrics                 | X                   |                    |                         | X
+ Spring Boot                        | X                   |                    |                         |  
+ Vue.js                             | X                   |                    |                         |  
+ Npm                                | X                   |                    |                         |  
+ OpenAPI                            | X                   |                    |                         |  
```

Comment:
They were left out **kind of** on purpose, but there's no real crystal clear logic behind. Old tech tables were also mixed bag, some technologies had versions (Akka 2.X), some did not (Logback, Liquibase), and there were some issues with the ones that listed versions (Why is PostgreSQL 10 listed using major version, but PostgreSQL 9.4 has minor version also? Why are some version numbers 9.X and some just "9"? RHEL8 seems to be missing?). So the numbering felt problematic and a bit messy and I wanted to avoid the whole issue.

I also though average reader is not too interested in things like Vue.js or Npm version, that this kind of detail does not necessarily need to be included (and updated) in this kind of "big picture" document.

But that is the backstory, looking at it now I don't think it is good. Versions are here and there, there is no consistency. So let's fix.

I am a bit tempted to
- (1) include major version number only if it is something else than the latest major release or if we support multiple versions (Java 8, Ubuntu 18 & 20)

Then there's alternative
- (2) include major version always (Systemd 237, Liquibase 3, Spring Boot 2)
but that sounds a bit noisy and technologies like PAM maybe don't need version numbers...

Or maybe
- (3) include major version always when version number is something we are directly controlling (Liquibase 3, Spring Boot 2, but "PAM" and "systemd")
 
Oh well, I guess I will go with (3). Or what do you think?

Reply:
I think (3) makes sense since the versions can be changed by someone who wants to build X-Road from the source up. The major version should be enough.

Reply:
Fixed, some fuzziness remains. Otherwise uses only major version, but Postgres 9.4 and Ubuntu 18.04 also have minor versions, since I guess they felt so significant.

Otherwise (3) is mostly used. 

Still the numbers feel a bit weird somehow in some cases, but maybe it's ok now.